$MirOS: ports/www/links+/patches/patch-md5hl_c,v 1.2 2005/05/23 22:22:46 tg Exp $
$OpenBSD: patch-md5hl_c,v 1.2 2003/09/23 06:35:33 fgsch Exp $
--- md5hl.c.orig	Thu Mar 24 00:47:00 2005
+++ md5hl.c	Thu Nov 16 22:59:43 2006
@@ -27,6 +27,10 @@
 
 #if !defined(HAVE_MD5INIT) || !defined(HAVE_MD5DATA) || !defined(HAVE_MD5_H)
 
+#include "cfg.h"
+
+#ifndef HAVE_MD5DATA
+
 #include "links.h"
 #include "md5.h"
 
@@ -79,6 +83,8 @@ MD5Data (const unsigned char *data, unsi
     MD5Update(&ctx,data,len);
     return MD5End(&ctx, buf);
 }
+
+#endif
 
 #endif
 
